app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -70,6 +70,11 @@ angular.module('Bastion.components').factory('Nutupane',
                 replace = replace || false;
                 table.working = true;
 
+                if (table.initialLoad) {
+                    table.refreshing = true;
+                    table.initialLoad = false;
```

Comment:
`initialLoad` itself isn't anew parameter -- it controls whether Nutupane makes a data call on initialization or not (so you can control more complex scenarios).